simplq/src/components/pages/QueueStatus.jsx

```diff
@@ -22,26 +23,45 @@ function QueueStatus() {
   const aheadCount = useSelector((state) => state.appReducer.aheadCount);
   const [updateInProgress, setUpdateInProgress] = useState(false);
 
-  const update = () => {
+  const showNotification = useCallback(() => {
+    const notificationImage = '/LogoLight.png';
+    const notificationText = `${queueName}: You've been notified by the queue manager.`;
```

Comment:
Since we're relying on the queue name to show notifications, this has a cascading effect across our hooks